src/adapter/price.ts

```diff
@@ -69,6 +69,8 @@ type IncludesMap = Record<string, Record<string, IncludeDetails>>
  * Data Feeds, its InputParameters must extend the basic ones (base/quote).
  */
 export class PriceEndpoint<T extends PriceEndpointGenerics> extends AdapterEndpoint<T> {
+  includesMap?: IncludesMap
```

Comment:
Noticed you kept the logic for includesMap in the PriceAdapter. Is this needed?

Reply:
removed